cmds/.gitignore

```diff
@@ -0,0 +1,59 @@
+# ls | sed 's:\(.*\):/\1/\1:' >> .gitignore
+/ansi/ansi
```

Comment:
Is there no way to specify all executables instead of doing this?

found this on stackoverflow:
http://stackoverflow.com/questions/5711120/gitignore-without-binary-files

```
# Ignore all
*

# Unignore all with extensions
!*.*

# Unignore all dirs
!*/

### Above combination will ignore all files without extension ###
```